src/Microsoft.AspNetCore.JsonPatch/breakingchanges.netcore.json

```diff
@@ -0,0 +1,7 @@
+[
+    {
+      "TypeId": "public interface Microsoft.AspNetCore.JsonPatch.Adapters.IObjectAdapter",
+      "MemberId": "System.Void Test(Microsoft.AspNetCore.JsonPatch.Operations.Operation operation, System.Object objectToApplyTo)",
+      "Kind": "Addition"
+    }
+  ]
```

Comment:
When you see something like this pop up - we need to discuss and get approval - or find an alternative.

Reply:
@Eilon knew about this. But I should look into the non-breaking change way of adding this you suggested above. Thanks